fdbclient/Knobs.cpp

```diff
@@ -232,6 +232,7 @@ void ClientKnobs::initialize(bool randomize) {
 	init( TAG_THROTTLE_RECHECK_INTERVAL,            5.0 ); if( randomize && BUGGIFY ) TAG_THROTTLE_RECHECK_INTERVAL = 0.0;
 	init( TAG_THROTTLE_EXPIRATION_INTERVAL,        60.0 ); if( randomize && BUGGIFY ) TAG_THROTTLE_EXPIRATION_INTERVAL = 1.0;
 
+	init( COMMIT_CLEAR_COST_ESTIMATE_METHOD,         0.0);
```

Comment:
This knob name doesn't make it clear that the value is a probability that the expensive estimation method will be chosen. It would be useful to get that information directly from the knob name.